### PR TITLE
Add BBC's privacy policy

### PR DESCRIPTION
**Pull request by Wingo5315**
**Type of pull request:** Product addition

**Related issues:** 

Provide more information about your pull request here. For help, see our contribution guide: http://privacyspy.org/contribute/

### CODE REVIEW

**Comment by ibarakaiev** [comment]
Hey, thanks for submitting this. There are syntactical issues in your `bbc.toml` file (notes and citations should be in an array). Please fix those and I'll take a look! Keep in mind that you should run `npm test` to check if everything works fine; you may also run `gulp` to see if the website is building properly on your local machine. Currently, `npm test` [fails](https://github.com/Politiwatch/privacyspy/pull/28/checks?check_run_id=821470155).

**Comment by milesmcc** [comment]
Thanks! Just adding on to what Igor said, it may be helpful to look at some other examples of TOML files in the `products` directory.

**Comment by Wingo5315** [comment]
No problem! I'll make those changes now.

File: products/bbc.toml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+citations = ["We have to have a valid reason to use your personal information. It's called the \'lawful basis for processing\'. Sometimes we might ask your permission to do things, like when you subscribe to an email. Other times, when you\'d reasonably expect us to use your personal information, we don't ask your permission, but only when the law says it\'s fine to use it, and it fits with the rights you have."]
+notes = ["The full list of why the BBC collects personal information is in Section 6 of the Privacy Policy."]
+
+[rubric.data-deletion]
+value = "yes-automated"
+citations = ["This depends on what information you're talking about. You can delete your account. Your account information is immediately deleted. Keep in mind we keep a record of how you've used our services, but this information can't be linked back to you (and) we also keep anything you\'ve uploaded or commented on. We might have collected other personal information that you might ask to be deleted that has nothing to do with BBC account."]
+
+[rubric.history]
+value = "last-modified"
+citations = [""]
```
Comment (ibarakaiev) [inline products/bbc.toml:27]: You have to include a citation here (such as the date it was last modified). PrivacySpy does not allow to have a rubric selection without either a note or a citation unfortunately.

File: products/bbc.toml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+[rubric.law-enforcement]
+value = "not-specified"
+citations = ["Sometimes by law we have to pass on your information to other organisations. We might also share your information if we have to by law, or when we need to protect you or other people from harm."]
+
+[rubric.list-collected]
+value = "generally"
+citations = ["We\'ll give you details about why we need your personal information and how we'll use it before you begin, unless it\'s obvious.
```
Comment (ibarakaiev) [inline products/bbc.toml:35]: If you want to include multi-line string, you have to include triple-quotes (see my suggestion).
```suggestion
citations = ["""We\'ll give you details about why we need your personal information and how we'll use it before you begin, unless it\'s obvious.
```

File: products/bbc.toml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+a. Information that you give us
+We might ask for your name and contact details, your date of birth or financial details, depending on what you're doing. If you apply to take part in a programme on politics, for example, we might ask you about your political opinions. Information you give us might include information in a creation you share with us, such as people's names that appear in the creation.
+When you register for a BBC account, or update your details or settings we ask for some personal information, like your email address and age.
+b. Device information
+Devices are lots of things like your computer, your mobile, your TV, your tablet and your voice-enabled device.
+We automatically collect some technical information from these devices and web browsers even when you\'re not signed into to a BBC account. This might include IP (internet protocol) address, device ID, app ID, vendor ID, advertising ID
+c. Information on your activities outside the BBC when you talk about us
+Like if you've mentioned us in a Twitter post, we\'ll collect your Twitter handle.
+d. Information on how you use our services
+Like the articles you read and the programmes you watch."]
```
Comment (ibarakaiev) [inline products/bbc.toml:45]: (as follows from the previous suggestion. FYI, I haven't tested it, but it _should_ work. See this helpful cheatsheet: https://toml.io/en/)
```suggestion
Like the articles you read and the programmes you watch."""]
```

File: products/bbc.toml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+value = "opt-out-all"
+citations = [""]
+
+[rubric.revision-notify]
+value = "yes"
+citations = ["We update this policy sometimes. If we make important changes, like how we use your personal information, we\'ll let you know. It might be a notice, an email or a message in your app."]
+
+[rubric.third-party-access]
+value = "yes-unspecified-critical"
+citations = ["We\'ll never sell your personal information. We do share it with others in these ways:
```
Comment (ibarakaiev) [inline products/bbc.toml:57]: Same here -- triple-quotes are required.

File: products/bbc.toml [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+e. When we do collaborative research
+We do research activities and sometimes collaborate with research partners. Every now and then we share our content and data with them. This might include information we\'ve collected about you. But we\'re careful about what we share and what our research partners can do with it.
+f. TV Licensing
+We share some personal information with TV Licensing, to check if you\'re using BBC iPlayer and to keep the licensing database accurate and up to date.
+g. Sometimes by law we have to pass on your information to other organisations
+We might also share your information if we have to by law, or when we need to protect you or other people from harm."]
+
+[rubric.third-party-collection]
+value = "yes"
+citations = [""]
```
Comment (ibarakaiev) [inline products/bbc.toml:75]: Again, you have to incorporate a citation or a note here.

**Comment by ibarakaiev** [comment]
Looks good—thanks for bearing with us! I am about to merge.

**Pull request merged**